src/pages/landing/stocks/_headerSection.js

```diff
@@ -0,0 +1,384 @@
+    }
+`
+const TopHeaderBgDiv = styled(Flex)`
+    width: 40%;
+    @media ${device.laptopM} {
+        width: 50%;
+    }
+    @media ${device.tablet} {
+        width: 100%;
+        justify-content: flex-end;
+        position: relative;
+    }
+`
+const TopHeaderImgWrapper = styled(Flex)`
+    align-items: flex-end;
+
+    @media ${device.laptop} {
+        max-width: 400px;
+    }
+    @media ${device.tabletL} {
+        max-width: 400px;
+    }
+    @media ${device.tablet} {
+        width: 275px;
+        position: absolute;
+        bottom: 0;
+        right: 0;
+        z-index: 2;
+    }
+    @media ${device.tabletS} {
+        width: 230px;
+    }
+    @media ${device.mobileL} {
+        width: 185px;
+    }
+`
+const SectionWrapper = styled(Flex)`
+    height: 720px;
+    padding: 0 120px;
+    flex-direction: row;
+    position: relative;
+    max-width: 1200px;
+    margin: 0 auto;
+    @media ${device.desktopL} {
+        max-width: 1600px;
+    }
+    @media ${device.laptopM} {
+        /* height: 625px; */
+    }
+    @media ${device.tabletL} {
+        /* height: 545px; */
+        padding: 32px 60px 27px;
+        flex-direction: column-reverse;
+    }
+    @media ${device.tabletS} {
+        padding: 32px 16px 62px;
+        height: auto;
+    }
+    @media ${device.mobileL} {
+        width: 90%;
+    }
+`
+const SectionImgWrapper = styled(Flex)`
+    width: 45%;
+    @media ${device.laptopM} {
+        align-items: center;
+    }
+    @media ${device.tabletL} {
+        width: 100%;
+        justify-content: center;
+    }
+`
+const SectionImgDiv = styled.div`
+    width: 100%;
+    position: relative;
+    z-index: 3;
+    @media ${device.tabletL} {
+        width: 100%;
+        max-width: 58.8rem;
+    }
+`
+const SectionBannerImg = styled.img`
+    width: 100%;
+    position: absolute;
+    top: -15%;
+    left: 0;
+    @media ${device.laptopM} {
+        position: unset;
+    }
+    @media ${device.tabletL} {
+        margin: 0 auto;
+    }
+`
+const SectionTextWrapper = styled(Flex)`
+    width: 55%;
+    align-items: center;
+    flex-direction: column;
+    margin-left: 120px;
+    @media ${device.laptopM} {
+        margin-left: 80px;
+    }
+    @media ${device.laptop} {
+        margin-left: 60px;
+    }
+    @media ${device.tabletL} {
+        width: 100%;
+        max-width: 58.8rem;
+        margin: 0 auto;
+    }
+`
+const SectionStyledHeader = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 6.4rem;
+        max-width: 60rem;
+        text-align: center;
+    }
+    @media ${device.tabletS} {
+        font-size: 4rem;
+        max-width: 60rem;
+        margin-bottom: 4px;
+    }
+`
+const SectionStyledHeaderSmall = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 2.4rem;
+        max-width: 100%;
+        text-align: center;
+    }
+    @media ${device.tabletS} {
+        font-size: 1.6rem;
+        max-width: 100%;
+    }
+`
+const is_ppc = true
```

Comment:
If `is_ppc` is always true, this should be removed

Reply:
removed as well in the latest commit..